person.rb

```diff
@@ -0,0 +1,38 @@
+require './nameable'
+require './decorators'
+
+# Person Class inherits from Nameable
+class Person < Nameable
```

Comment:
- [ ] OPTIONAL
This activity does not require the _Person_ class inherit from a Nameable class. Kindly improve upon the same and remove enrequired methods from the _Person_ class.